docs/pages/desktop-access/active-directory.mdx

```diff
@@ -1,158 +1,173 @@
-      31qA4dO3if7RdikD9hVbiIF9jQ==
-      -----END CERTIFICATE-----
-
-  discovery:
-    base_dn: '*'
-  labels:
-    teleport.internal/resource-id: 42d8859c-60d0-4d7f-9767-bdd66b63fce6
-```
-
-Install the Teleport Desktop Service on your Linux host.
-
-(!docs/pages/includes/install-linux.mdx!)
-
-In the Teleport Web UI, click **Next**.
-
-## Step 2/2. Start Teleport
-
-Once you've saved `/etc/teleport.yaml`, start Teleport:
-
-(!docs/pages/includes/start-teleport.mdx service="the Teleport Desktop Service"!)
-
-The access wizard will detect when the new Teleport instance has joined the
-cluster, and you can then click **Next**.
-
-Teleport will discover available Windows desktops in the domain, and list them
-under **Desktops**. Click **Finish** then **BROWSE EXISTING RESOURCES** to see them:
-
-![Desktops Discovered](../../img/desktop-access/auto-discovered-desktops.png)
-
-## Troubleshooting
+- Access to a running Teleport cluster, `tctl` admin tool, and `tsh` client tool 
+  version >= (=teleport.version=). 
+  
+  You can verify the tools you have installed by running the following commands:
 
-If you hit any issues, check out the
-[Troubleshooting documentation](./troubleshooting.mdx) for common problems and
+  ```code
+  $ tctl version
+  # Teleport Enterprise v(=teleport.version=) go(=teleport.golang=)
+  
+  $ tsh version
+  # Teleport v(=teleport.version=) go(=teleport.golang=)
+  ```
+
+  You can download these tools by following the appropriate [Installation 
+  instructions](../installation.mdx#installation-instructions) for your environment 
+  and Teleport edition.
+
+- A server or virtual machine running the Windows Server operating system on which you can
+  install Active Directory services.
+
+- A Linux host to run the Teleport Windows Desktop Service.
+  You can use an existing server that runs Teleport for other resources.
+
+## Step 1/3. Install and configure Active Directory
+
+The Teleport Web UI provides scripts to automate the configuration of Active Directory
+for the Teleport Windows Desktop Service. From the Teleport Web UI, you can 
+download the following scripts to convert a Windows Server into an Active Directory
+domain controller:
+
+- The Active Directory installation script adds basic Active Directory services and tools to the 
+  Windows Server.
+- The Active Directory Certificate Services installation script adds a certificate authority and 
+  certificate management services to the domain controller.
+- The Active Directory configuration script generates Teleport configuration settings for your specific
+  cluster.
+
+Installing Active Directory Certificate Services is optional. However, Teleport requires secure LDAP 
+connections for certificate-based authentication, and LDAP traffic is not encrypted by default. 
+To provide secure LDAP connections (LDAPS), you must have a certificate authority that issues properly 
+formatted and encrypted certificates. By installing the [Active Directory Certificate 
```

Comment:
I don't think we need to direct the reader to an external docs site here. I would suggest explaining Active Directory Certificate Services in one sentence in this paragraph instead.